docs/products/metrics/howto/storage-overview-page.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Storage overview for Aiven for Metrics
+sidebar_label: Storage overview page
+---
+Get complete insights into your storage usage with Aiven for Metrics. This overview page offers data that helps you better manage and understand your object storage usage and associated costs. Access important metrics such as current and forecasted billing, object storage consumption, and settings for data retention and storage threshold alerts.
```

Comment:
"this overview page" is unclear to me, I thought you were talking about this very docs page, but I'm thinking you might be talking about the UI page? 

"Storage overview page" is also not a very "how-to" compatible title, is there something we can do about the TOC or title? Let's talk about this kind of situations in our next team sync 👍

Reply:
Thanks for pointing that out, Arthur! I've updated the title to "Manage Storage" and clarified the term to "Storage" page in the Aiven Console to avoid any confusion. Let’s discuss more in our next sync to refine our approach further. Appreciate your feedback!